charmcraft.yaml

```diff
@@ -23,13 +23,13 @@ resources:
   oci-image:
     type: oci-image
     description: OCI image for login-ui
-    upstream-source: ghcr.io/canonical/identity-platform-login-ui:v0.15.0
+    upstream-source: ghcr.io/canonical/identity-platform-login-ui:v0.16.0
 
 requires:
   ingress:
     interface: ingress
-  kratos-endpoint-info:
-    interface: kratos_endpoints
+  kratos-info:
+    interface: kratos_info
```

Comment:
It is not a breaking change considering that we use bundle versioning. [This](https://github.com/canonical/iam-bundle/blob/66891eb99127b3b17b4f859c0cdc2c2604166346/bundle.yaml.j2#L77) will need to be updated when the login ui rev is bumped. If you upgraded the charm in an existing deployment, you'd need to remove the old `kratos-endpoint-info` relation and add the new `kratos-info`.
Technically we could keep the old relation name to avoid having to do it, it just wouldn't fully reflect the scope of the interface - what do you think?

Reply:
not sure in all fairness, fact we have no users means we are not in need of being really tight with the releases right now so happy to forge ahead with the change